crate/operator/main.py

```diff
@@ -382,191 +374,59 @@ async def cluster_create(
 
 
 @kopf.on.update(API_GROUP, "v1", RESOURCE_CRATEDB)
-async def cluster_update(
+async def update(
```

Comment:
Wouldn't it make sense to have something in the name of this function that says that this function is for handling updates on custom CrateDB resources? 

Something like `update_crate_db` or `update_crate_db_resource`? 🤔

Reply:
I guess `cluster_update` makes sense, doesn't it? That's then in sync with `cluster_create()` with `@kopf.on.create`.

Reply:
Yeah, it also makes sense. 👍